### ISSUE TITLE
Maps repo

### ISSUE THREAD

**Issue by IhateTrains** [IhateTrains/EBS#1]
A bunch of files sent to me by -Incorrectish from the Paradox forum
- 363 political map
![image](https://user-images.githubusercontent.com/29546927/47260240-bc6a8000-d4b7-11e8-8906-4a81052024fd.png)

- religious map
![image](https://user-images.githubusercontent.com/29546927/47260246-db691200-d4b7-11e8-90a2-ff0cf6d8767c.png)

> Brown-buddhist
dark brown african pagan
peach east african pagan
light brown baltic pagan
green hindu
dark green zoroastrian
light blue -jewish
purple nicene
red-germanic
grey - tengri
light red semetic
dark gray ashurian
pink celtic
light green slavic
way light blue bon
light purple finnic

- blank map used by him
![image](https://user-images.githubusercontent.com/29546927/47260253-fdfb2b00-d4b7-11e8-9918-d72ab4c2caab.png)

Note that we won't replace the current map with this map, it's just for showing how the realm and religion setup **could** look like (it e.g. can be wrong in some places).

**Comment by tannerflick4** [comment]
What implications does this have? Anything that would be changed?

**Comment by IhateTrains** [comment]
Changed in the religion setup? More Hellenism than in 476.

**Comment by TheProgrammingProject** [comment]
Is Armenia independent on this map? Also,where’s bon?

**Comment by IhateTrains** [comment]
Yes, Armenia should be independent in 363 according to https://en.m.wikipedia.org/wiki/Kingdom_of_Armenia_(antiquity).
Bon is the lightest blue on the second map.

**Comment by TheProgrammingProject** [comment]
Ah,Thanks.Will note this information for the Julian bookmark.Also,what religion is the red in Africa?

**Comment by loup99** [comment]
![romana235ad](https://user-images.githubusercontent.com/7620180/47263602-fada5c00-d504-11e8-9021-fab432398ada.jpg)

![romana271](https://user-images.githubusercontent.com/7620180/47263608-03329700-d505-11e8-9b6f-94c1dc53d8e0.jpg)

![romana305ad](https://user-images.githubusercontent.com/7620180/47263609-075eb480-d505-11e8-8a59-23b6e1dcec88.jpg)

![romana306ad](https://user-images.githubusercontent.com/7620180/47263614-10e81c80-d505-11e8-87d0-6478e36c17f9.jpg)

![romana311ad](https://user-images.githubusercontent.com/7620180/47263616-15acd080-d505-11e8-9096-cdb8de73e6e0.jpg)

![romana313ad](https://user-images.githubusercontent.com/7620180/47263619-19d8ee00-d505-11e8-9101-98a102408a63.jpg)

![romana340ad](https://user-images.githubusercontent.com/7620180/47263622-1fcecf00-d505-11e8-8f9d-04b7b020c9d3.jpg)

![romana317ad](https://user-images.githubusercontent.com/7620180/47263623-1fcecf00-d505-11e8-877f-478215f4a64c.jpg)

![romana335ad](https://user-images.githubusercontent.com/7620180/47263624-20676580-d505-11e8-8f17-ddb0c4bc3542.jpg)

![romana337](https://user-images.githubusercontent.com/7620180/47263625-20676580-d505-11e8-9099-850e2ce13c55.jpg)

![romandioceses305s](https://user-images.githubusercontent.com/7620180/47263630-3c6b0700-d505-11e8-824c-1cf6fbf45c9a.jpg)

![rome312](https://user-images.githubusercontent.com/7620180/47263631-412fbb00-d505-11e8-8c8d-2f45516ffd59.jpg)

![caucasia300s](https://user-images.githubusercontent.com/7620180/47263634-4856c900-d505-11e8-8892-941f1494d748.jpg)

**Comment by loup99** [comment]
![image](https://user-images.githubusercontent.com/7620180/47263655-00847180-d506-11e8-9fbf-e438f50de302.png)

![image](https://user-images.githubusercontent.com/7620180/47263657-05492580-d506-11e8-95f8-c9a6503f4a83.png)

**Comment by loup99** [comment]
![image](https://user-images.githubusercontent.com/7620180/47263719-6f15ff00-d507-11e8-9f31-afa5d863e55e.png)

![image](https://user-images.githubusercontent.com/7620180/47263717-645b6a00-d507-11e8-9218-184a6384bf15.png)

![image](https://user-images.githubusercontent.com/7620180/47263722-8a810a00-d507-11e8-9802-cf7828b05a79.png)

![image](https://user-images.githubusercontent.com/7620180/47263728-a5ec1500-d507-11e8-9d04-8a78213498d8.png)

![image](https://user-images.githubusercontent.com/7620180/47263731-b00e1380-d507-11e8-86f4-619289eaaa1f.png)

**Comment by IhateTrains** [comment]
Thanks for all these maps 👍

**Comment by loup99** [comment]
You are welcome, although you might want to change the title of the issue to be about maps of the pre-395 period in general since I kind of highjacked the original conversation.

**Comment by IhateTrains** [comment]
@TheProgrammingProject according to the legend the colour is peach and represents east african pagan.
What I noticed is that there is no Kemetic on this map.

**Comment by loup99** [comment]
@IhateTrains If I were you I would be very careful with using that map at all, because there are quite a lot of oddities that strike me immediately as rather odd since they contradict academic work made for the period. (not on Kemetic in particular however, that I have no knowledge of)

**Comment by TheProgrammingProject** [comment]
@loup99 Thanks for the answer.I actually think the red I was talking about (right under the blue and purple and above the dark brown) is Kemetic.

**Comment by IhateTrains** [comment]
@TheProgrammingProject it's east_african_pagan, which is a different religion.

**Comment by TheProgrammingProject** [comment]
@IhateTrains Ok,I thought east_african_pagan was the yellowish color under the Jewish in Africa.

**Comment by IhateTrains** [comment]
Shit, I'm colorblind. The religion you originally asked about is not in the legend I guess.
The map author could make it both Kemetic and Central African.

**Comment by IhateTrains** [comment]
@loup99 I know this map shouldn't be trusted, don't worry.

**Comment by IhateTrains** [comment]
![image](https://media.discordapp.net/attachments/335082344346288130/569843818866802689/350px-Rechila.gif)
